man/fi_rxm.7.md

```diff
@@ -90,13 +90,18 @@ The ofi_rxm provider checks for the following environment variables.
 *FI_OFI_RXM_BUFFER_SIZE*
 : Defines the transmit buffer size / inject size. Messages of size less than this
   would be transmitted via an eager protocol and those above would be transmitted
-  via a rendezvous protocol. Transmit data would be copied up to this size
-  (default: ~16k).
+  via a rendezvous or SAR (Segmentation And Reassembly) protocol. Transmit data
+  would be copied up to this size (default: ~16k).
 
 *FI_OFI_RXM_COMP_PER_PROGRESS*
 : Defines the maximum number of MSG provider CQ entries (default: 1) that would
   be read per progress (RxM CQ read).
 
+*FI_OFI_RXM_SAR_LIMIT*
+: Set this environment variable to control the RxM SAR (Segmentation And Reassembly)
+  protocol. Messages of size less (greater) than this (default: 256 Kb) would be
+  transmitted via eager (rendezvous) protocol.
```

Comment:
The wording isn't quite right.  Messages larger than the SAR size are transmitted using rendezvous.  The SAR_LIMIT isn't directly linked to the eager protocol, as that's controlled using the RXM_BUFFER_SIZE.  I.e. SAR is used when the size is > BUFFER_SIZE but <= SAR_LIMIT.

Reply:
Oops, sorry. I wrote this at night :)
The wording was paraphrased